### ISSUE TITLE
OpenSSL CVE-2015-1793 - Alternative chains certificate forgery

### ISSUE THREAD

**Issue by ewindisch** [gliderlabs/docker-alpine#50]
see: https://github.com/docker-library/official-images/issues/878

**Comment by ewindisch** [comment]
Update applied in alpine: http://git.alpinelinux.org/cgit/aports/commit/?id=4cc1178d550587aaaf640ef084feeadb2e751b80

**Comment by ewindisch** [comment]
Actually, this should not affect the images currently on Docker Hub as they were built prior to the release of the vulnerable versions of OpenSSL.

**Comment by jumanjiman** [comment]
it affects alpine:3.1 image

```console
user@devenv:~$ docker run --rm -it alpine:3.1 sh
/ # apk update
fetch http://dl-4.alpinelinux.org/alpine/v3.1/main/x86_64/APKINDEX.tar.gz
v3.1.4-65-g319998e [http://dl-4.alpinelinux.org/alpine/v3.1/main]
OK: 4864 distinct packages available

/ # apk search openssl
openssl-1.0.1o-r0
py-openssl-0.14-r0
perl-crypt-openssl-rsa-0.28-r3
openssl-dev-1.0.1o-r0
openssl-doc-1.0.1o-r0
perl-crypt-openssl-random-0.06-r1
perl-crypt-openssl-rsa-doc-0.28-r3
php-openssl-5.6.10-r0
perl-crypt-openssl-random-doc-0.06-r1
libevent-2.0.21-r2
dovecot-2.2.15-r1
acf-openssl-0.9.0-r0
```

**Comment by ewindisch** [comment]
Thanks @jumanjiman

**Comment by toli** [comment]
Seems that this CVE has been fixed in Alpine `3.2.2`: http://www.alpinelinux.org/posts/Alpine-3.2.2-released.html

would be nice if there was an update to have `3.2.2` version available as a Docker image

**Comment by andyshinn** [comment]
I'll get a pull request open tomorrow to the official images repo to bump the base packages for this CVE.

**Comment by andyshinn** [comment]
This is closed by #67 and docker-library/official-images#1050.
